src/StereoDB/Infra/Sql/Sql.fs

```diff
@@ -99,6 +99,13 @@ module internal QueryBuilder =
                         if columnProperty <> null then
                             Some columnProperty
                         else None)
+        
+        member this.GetTableColumn tableName columnName = 
```

Comment:
better use F# Fantomas formatter:
member this.GetTableColumn(tableName, columnName) =